### PR TITLE
Allows basil and sybil captains to yell at eachother

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Via comms console. Have obviously not tested yet.

:cl: Kor
rscadd: The comms console now has a new option allowing you to communicate with the other server.
/:cl:

### CODE REVIEW

**Comment by ike709** [comment]
inb4 we'll be able to travel between sybil and basil space stations with a shuttle

**Comment by as334** [comment]
I'm actually so excited

**Comment by PKPenguin321** [comment]
i can't wait to join both servers as captain and talk to myself

**Comment by Aloraydrel** [comment]
War of the station event soon

**Comment by MMMiracles** [comment]
inb4 bee script is sent to the other station

**Comment by KorPhaeron** [comment]
This is probably just gonna end with sybil spamming memes at basil yeah

**Comment by coiax** [comment]
This is going to be weird, given the rounds aren't gonna be in sync

**Comment by KorPhaeron** [comment]
Obviously we need news reports as other stations start and end.

"Fat Camp Zulu has been destroyed by a nuclear blast. Donk Co. denies involvement."

**Comment by Saegrimr** [comment]
>BAGIL SEND HELP WE GOT REVS
>4 braindeads on bagil, 4 new latejoin assistants on sybil

Gonna need to think about how IC in OOC this ends up being.

**Comment by KorPhaeron** [comment]
If I die on sybil I will simply hop onto basil and send a message with my killers name

**Comment by coiax** [comment]
Asking for assistance and a bunch of assistants turning up is probably okay.
Being dead and then transfering that OOC information IC on a sister server probably not

File: code/game/machinery/computer/communications.dm [hunk 1]
```diff
@@ -116,6 +116,19 @@ var/const/CALL_SHUTTLE_REASON_LENGTH = 12
 			else if (src.authenticated==2 && message_cooldown)
 				usr << "Intercomms recharging. Please stand by."
 
+		if("crossserver")
+			if(src.authenticated==2)
```
Comment (Fox-McCloud) [inline code/game/machinery/computer/communications.dm]: implied `src`

File: code/game/machinery/computer/communications.dm [hunk 2]
```diff
@@ -116,6 +116,19 @@ var/const/CALL_SHUTTLE_REASON_LENGTH = 12
...
 
+		if("crossserver")
+			if(src.authenticated==2)
+				if(CM.lastTimeUsed + 600 > world.time)
+					usr << "Arrays recycling.  Please stand by."
+					return
+				var/input = stripped_input(usr, "Please choose a message to transmit to an allied station.  Please be aware that this process is very expensive, and abuse will lead to... termination.", "Send a message to an allied station.", "")
+				if(!input || !(usr in view(1,src)))
+					return
+				send2otherserver(world.name, input,"Comms_Console")
```
Comment (Iamgoofball) [inline code/game/machinery/computer/communications.dm]: where is this proc

File: code/modules/admin/verbs/adminhelp.dm [hunk 3]
```diff
@@ -154,7 +154,8 @@
 /proc/send2otherserver(source,msg,type = "Ahelp")
```
Comment (KazeEspada) [inline code/modules/admin/verbs/adminhelp.dm]: right here you goof

Comment (Iamgoofball) [inline code/modules/admin/verbs/adminhelp.dm]: your mom is so fat

**Comment by Jordie0608** [comment]
Changelog

**Comment by Bawhoppen** [comment]
should be able to send cargo credits to the other server too

File: code/game/machinery/computer/communications.dm [hunk 4]
```diff
@@ -116,6 +116,19 @@ var/const/CALL_SHUTTLE_REASON_LENGTH = 12
 			else if (src.authenticated==2 && message_cooldown)
 				usr << "Intercomms recharging. Please stand by."
 
+		if("crossserver")
+			if(src.authenticated==2)
+				if(CM.lastTimeUsed + 600 > world.time)
+					usr << "Arrays recycling.  Please stand by."
+					return
+				var/input = stripped_input(usr, "Please choose a message to transmit to an allied station.  Please be aware that this process is very expensive, and abuse will lead to... termination.", "Send a message to an allied station.", "")
```
Comment (Jordie0608) [inline code/game/machinery/computer/communications.dm]: You probably want to use stripped_multiline_input as it's easier to write long messages with.

**Pull request merged**